src/components/Article/Article.vue

```diff
@@ -43,8 +44,9 @@ export default {
       var formattedDate = (date.getMonth() + 1) + '/' + date.getDate() + '/' + date.getFullYear() + '  ' + strTime
       return formattedDate.slice(0, 8)
     },
-    openArticle() {
-      this.$router.push('/post/' + 'defaultarticle')
+    openArticle(id, post) {
+      // this.$router.push({name: 'post', path: '/post/' + id, params: {eventId: id}})
+      this.$router.push({name: 'post', params: {eventId: id, article: post[0]['content']}})
```

Comment:
A more structured prop can be passed once we setup the `content` field with the second exchange post format